scripts/bottle-hashes.sh

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env bash
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+# This script takes a directory where the bottles are stored as its argument.
+# Run it from the base directory (tezos-packaging).
+
+if [[ -d ./Formula ]]
+then
+    if [[ -d "$1" ]]
+    then
+        regex="(tezos-.*)-v[0-9]+\.[0-9]+-[0-9]+\.(catalina|mojave)\.bottle\.tar\.gz"
```

Comment:
will `rc`s match rhis regex? :thinking:

Reply:
to add to the comment above: we know for sure (according to the current tezos naming scheme) that the `v` is also used at the beginning of every version name.

Consequently what's between the last `-v` before the `.$os`  and the latter iself is all part of the version, which isn't something we are actually interested in, for this script.

I'd suggest a more general expression to gloss over it, I think
```suggestion
        regex="(tezos-.*)-v.*\.(catalina|mojave)\.bottle\.tar\.gz"
```
should work.